src/PrestaShopBundle/Twig/HookExtension.php

```diff
@@ -62,7 +62,7 @@ class HookExtension extends AbstractExtension
     public function __construct(
         HookDispatcherInterface $hookDispatcher,
         ModuleDataProvider $moduleDataProvider,
-        ModuleRepository $moduleRepository = null
+        ModuleRepositoryInterface $moduleRepository = null
```

Comment:
```suggestion
        ?ModuleRepositoryInterface $moduleRepository = null
```

Reply:
Actually `$moduleRepository` dependency is mandatory and shouldn't be optional. Take a look at line 138.
```suggestion
        ModuleRepositoryInterface $moduleRepository
```

Reply:
I agree it seems weird, and yet when you look at the service definition it is consistent:
https://github.com/PrestaShop/PrestaShop/blob/88bce71fb5def465a09d57bf22f20c62ede057a5/src/PrestaShopBundle/Resources/config/services/bundle/twig.yml#L53

Reply:
Perhaps because if there is no employee in context, then there is no HTML to render ?
Looks like pure luck :))